src/components/PickupBox/PickupBox.jsx

```diff
@@ -0,0 +1,171 @@
+import { React, useState } from 'react';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionPanel,
+  AccordionButton,
+  AccordionIcon,
+  ChakraProvider,
+  FormControl,
+  FormLabel,
+  Input,
+  Textarea,
+} from '@chakra-ui/react';
+
+import { BsFillCheckCircleFill, BsXCircleFill } from 'react-icons/bs';
+import PropTypes from 'prop-types';
+import styles from './PickupBox.module.css';
+import RejectBoxPopup from '../AlertPopups/RejectBoxPopup/RejectBoxPopup';
+import PickupBoxIcon from '../BoxIcons/PickupBoxIcon.svg';
+import ImageVector from '../BoxIcons/ImageVector.svg';
+import { FYABackend } from '../../common/utils';
+
+const PickupBox = ({
+  approved,
+  boxID,
+  boxHolderName,
+  boxHolderEmail,
+  zipCode,
+  picture,
+  date,
+  status,
+  rejectionReason,
+  fetchBoxes,
+  pickup,
+}) => {
+  // A state for determining whether or not the rejectBoxPopup is open
+  // This state is set true when the reject button is clicked
+  const [rejectBoxPopupIsOpen, setRejectBoxPopupIsOpen] = useState(false);
+
+  // A function that updates the approved boolean in the backend and refreshes all boxes that are under review
+  // This method is called when the approve box icon is clicked
+  const approvePickupBox = async id => {
+    FYABackend.put('/boxHistory/approveBox', {
+      boxID: id,
+    }).then(async () => {
+      await fetchBoxes('under review', true);
+    });
+  };
+
+  return (
+    <ChakraProvider>
+      <div
+        className={`${styles.box}
+        ${status === 'evaluated' && approved === true ? styles['box-approved'] : ''}
+        ${status === 'evaluated' && approved === false ? styles['box-rejected'] : ''}`}
+      >
+        <Accordion allowToggle>
+          <AccordionItem>
+            {/* Pickup box ID and date */}
+            <h3>
+              <AccordionButton className={styles['accordion-button']}>
+                <img src={PickupBoxIcon} alt="" />
+                <div className={styles['title-div']}>
+                  <p className={styles.title}>
+                    <p className={styles['box-number']}>Box #{boxID}</p>
+                    {date}
+                  </p>
+                </div>
+                <div className={styles['arrow-button']}>
+                  <AccordionIcon />
+                </div>
+              </AccordionButton>
+            </h3>
+            {/* Box details */}
+            <AccordionPanel className={styles['accordion-panel']} pb={4}>
+              <div className={styles['box-details']}>
+                {picture !== null && (
+                  <img src={picture} alt="" className={styles['pickup-image-corners']} />
+                )}
+                {picture === null && status !== 'evaluated' && (
+                  <div className={styles['image-wrapper']}>
+                    <img className={styles['image-box']} src={ImageVector} alt="" />
+                  </div>
+                )}
```

Comment:
If there is no submitted picture, then we do not want to display the picture section. So we can remove the ImageVector entirely (including the SVG).

Reply:
I remember at the meeting with the admin, they said they wanted to be able to delete and upload pictures within the admin approval process. Are we going to implement this feature?

Reply:
removed the picture section nonetheless

Reply:
IIRC I believe they just wanted the ability to reject an image, but not uploading a new image.